packages/api/src/services/actions/emp.ts

```diff
@@ -32,7 +32,7 @@ export function Handlers(config: Config, appState: Dependencies): Actions {
       return Array.from(registeredEmps.values());
     },
     lastBlock() {
-      return appState.lastBlock;
+      return appState.lastBlockUpdate;
```

Comment:
What's the difference between lastBlock and the pre-existing lastBlockUpdate?

Reply:
in the previous code, we maintained a var for the last block seen based on block events (lastBlock). There was a separate counter for the block we ran a state update on, ie it would run every 100 blocks (lastBlockUpdate).  Its confusing, but now that we dont use block events, we can remove one. 

Also i mainly have this call here for debugging so i can check now how far behind our last update was through the api.